gh-pages/content/compiler-and-rosetta-maintenance.md

```diff
@@ -0,0 +1,142 @@
+    * [TypeScript 4.7 release notes](https://devblogs.microsoft.com/typescript/announcing-typescript-4-7/)
+    * [TypeScript 4.8 release notes](https://devblogs.microsoft.com/typescript/announcing-typescript-4-8/)
+    * [TypeScript 4.9 release notes](https://devblogs.microsoft.com/typescript/announcing-typescript-4-9/)
+    * [TypeScript 5.0 release notes](https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/)
+
+In addition to these, two import changes affect jsii exported APIs specifically:
+
+* The `1.x` release line silently ignores index signatures (these are hence unavailable in other languages), and the
+  `5.0.x` release starts explicitly rejecting these
+    * You can explicitly opt into the `1.x` behavior by adding the `@jsii ignore` doc-tag:
+      ```ts hl_lines="5"
+      export interface Example {
+        /**
+         * This API is not visible in non-TS/JS languages.
+         *
+         * @jsii ignore
+         */
+        readonly [key: string]: any;
+      }
+      ```
+
+* The `1.x` release line incorrectly interprets tuple types as synonyms to `object` (resulting in those APIs often being
+  unusable from other languages), and the `5.0.x` release starts explicitly rejecting these
+    * These APIs need to be replaced. You can use the  `@jsii ignore` doc-tag to explicitly opt these APIs out of the
+      multi-language supported API, and provide an alternate API:
+      ```ts hl_lines="3-4 8"
+      export class Example {
+          /**
+           * @jsii ignore
+           * @deprecated Prefer using `newMethod` instead.
+           */
+          public method(): [string, number] { /* ... */ }
+
+          public newMethod(): StringAndNumber { /* ... */ }
+      }
+
+      export interface StringAndNumber {
+          readonly stringValue: string;
+          readonly numberValue: number;
+      }
+      ```
+
+### What happens with other packages of the jsii toolchain (`jsii-pacmak`, `jsii-config`, etc...)?
+
+The new versioning strategy only affects the `jsii` and `jsii-rosetta` packages. All other parts of the jsii toolchain
+will continue to be released under the `1.x` release line for the foreseeable future. The compilation artifacts produced
+by `jsii@5.0.0` and newer remain compatible with jsii tools from the `1.x` toolchain, so developers need not coordinate
+upgrades with their dependents and dependencies.
+
+### If I upgrade my package to `jsii@5.0.0`, are my dependents required to do the same?
+
+You can decide to upgrade your `jsii` compiler as well as `jsii-rosetta` independently from your dependencies and
+dependents. Output artifacts are compatible across all tool releases, including the `1.x` line, at least until they
+reach *End-of-Support*.
+
+### Can my app have dependencies built by different `jsii` release lines?
+
+The `jsii` compiler and `jsii-rosetta` versions used to build a library has no material impact on the runtime artifacts.
```

Comment:
```suggestion
The `jsii` compiler and `jsii-rosetta` versions used to build a library which has no material impact on the runtime artifacts.
```

Reply:
I think this changes the meaning of the sentence (which is there to say your choice of compilers doesn't affect your consumers).